infrastructure/docs/logs.md

```diff
@@ -36,11 +56,22 @@ To view all of the logs for the doc-index-updater:
 stern -n doc-index-updater doc-index-updater
 ```
 
-See the [stern docs][2] for more info.
+A useful command for viewing the logs related to a specific correlation id
+would be to use the `-i` or `--include` parameter with a regex
+which looks for `INFO`, `WARN`, and `ERROR` messages matching that id.
+
+See here, for `b9912c04-325e-4f1b-8943-3ef319a88989`:
 
-### Viewing container logs in Azure
+```sh
+stern doc-index-updater -n doc-index-updater -i 'level.:.(INFO|WARN|ERROR).*b9912c04-325e-4f1b-8943-3ef319a88989'
+```
 
-[`stern`][2] is the best way to view container logs, although you can view the container logs in the Logs Analytics workspace if you wish.
```

Comment:
I've found it really useful to be able to pull data out of the JSON log entries via the Azure UI, which I don't think is possible using `stern`. Calling stern "the best way" was wrong IMO.